infra/azure/variables.tf

```diff
@@ -24,3 +24,14 @@ variable k8s_machine_type {
   type = string
   default = "Standard_D2_v2"
 }
+
+variable "ci_config" {
+  type = object({
+    github_oauth_token = string
+    github_user1_oauth_token = string
+    watched_branches = list(tuple([string, bool, bool]))
+    deploy_steps = list(string)
+    github_context = string
+  })
+  default = null
+}
```

Comment:
It feels odd to me that these duplicate the variables in ci. What's your plan for these two things?